Shared/LocalStorageManager.swift

```diff
@@ -59,7 +59,7 @@ private extension LocalStorageManager {
             container.persistentStoreDescriptions.first!.url = sharedStoreURL
         }
 
-        container.loadPersistentStores { description, error in
+        container.loadPersistentStores { _, error in
```

Comment:
Since `description` is unused in the closure, we can replace it with a `_` placeholder in the capture list.